src/chi/components/data-table/data-table.scss

```diff
@@ -0,0 +1,721 @@
+@import '_variables';
+@import '_mixins';
+
+$sizes: (
+  xs: (
+    padding: 0.25rem,
+  ),
+  sm: (
+    padding: 0.5rem,
+  ),
+  md: (
+    padding: 0.75rem,
+  ),
+  lg: (
+    padding: 1rem,
+  ),
+  xl: (
+    padding: 1.25rem,
+  )
+);
+
+.chi-data-table {
+  border-collapse: collapse;
+  display: block;
+  font-size: $font-size-base;
+  line-height: $line-height-sm;
+  overflow-x: auto;
+  overflow-y: hidden;
+  width: 100%;
+
+  @include respond-to(md) {
+    display: flex;
+    flex-direction: column;
+  }
+
+  label {
+    margin-bottom: 0;
+  }
+
+  @each $type in map-keys($sizes) {
+    &.-#{$type} {
+      .chi-data-table__head {
+        .chi-data-table__cell {
+          padding: map-get(map-get($sizes, $type), padding);
+        }
+      }
+
+      .chi-data-table__body {
+        .chi-data-table__cell {
+          padding: map-get(map-get($sizes, $type), padding);
+        }
+      }
+    }
+  }
+
+  &__head {
+    display: none;
+
+    @include respond-to(md) {
+      background-color: $color-background-white;
+      display: block;
+    }
+
+    .chi-data-table__cell {
+      @include respond-to(md) {
+        border-bottom: 0.125rem solid $color-border-base-dark;
+        display: flex;
+        font-weight: $font-weight-semi-bold;
+        padding: 0.75rem;
+        position: relative;
+
+        .chi-icon {
+          margin-left: 0.5rem;
+
+          &.icon-arrow-sync {
+            color: $color-icon-muted-light;
+            transform: rotate(90deg);
+            transition: transform 0.2s ease-in-out;
+          }
+        }
+      }
+    }
+  }
+
+  &__body {
+    @include respond-to(md) {
+      display: flex;
+      flex-direction: column;
+    }
+
+    .chi-data-table__cell {
+      display: flex;
+      padding: 0.25rem 0.75rem;
+
+      @include respond-to(md) {
+        border-bottom: 0.0625rem solid $color-border-base-light;
+        padding: 0.75rem;
+        word-break: break-word;
+      }
+
+      .-text--truncate {
+        cursor: pointer;
+      }
+
+      &::before {
+        content: attr(data-label);
+        display: block;
+        font-weight: $font-weight-semi-bold;
+        min-width: 10rem;
+
+        @include respond-to(md) {
+          display: none;
+        }
+      }
+    }
+  }
+
+  &__footer {
+    display: none;
```

Comment:
This is hiding all pagination on mobile. Would recommend removing. If there's a different pagination pattern on mobile that portal is proposing it will have to wait for phase 2.